src/doc/rustc/src/json.md

```diff
@@ -229,6 +229,8 @@ flag][option-emit] documentation.
        - "link": The generated crate as specified by the crate-type.
        - "dep-info": The `.d` file with dependency information in a Makefile-like syntax.
        - "metadata": The Rust `.rmeta` file containing metadata about the crate.
+       - "asm": The `.s` file with generated assembly
+       - "llvm_ir": The `.ll` file with generated textual LLVM IR
```

Comment:
For symmetry adding the `obj` kind would make sense to me.

Reply:
Also the current description reads like there can be only one file of the asm and llvm_ir kinds.

Reply:
> For symmetry adding the obj kind would make sense to me.

Sure, will look into adding that as well as all other file types listed [here](https://doc.rust-lang.org/rustc/command-line-arguments.html#--emit-specifies-the-types-of-output-files-to-generate)